### ISSUE TITLE
Returning JSON

### ISSUE THREAD

**Issue by humblebeee** [gateio/gateapi-python#116]
Is there any way to make the client return JSON objects instead of Ticker for the `client.list_tickers()` method?

**Comment by revilwang** [comment]
Yes, you can pass `_preload_content=False` when you call any API method like `list_tickers()` to avoid JSON response deserialization.

**Comment by humblebeee** [comment]
I wish this was mentioned in the documentation, nevertheless thank you!

**Comment by revilwang** [comment]
Actually the [method python doc](https://github.com/gateio/gateapi-python/blob/master/gate_api/api/spot_api.py#L441) tells you that

**Comment by humblebeee** [comment]
I didn't notice! Probably because the naming is not intuitive and my IDE lists parameters with preceding `_` at the end (maybe because of a coding convention in Python!)

**Comment by revilwang** [comment]
Problem solved no matter what. Issue closed.

**Comment by humblebeee** [comment]
Yes, thank you! On a side note: A comprehensive documentation would be very much appreciated and saved so much time preventing issues being created.
